arrow-libs/core/arrow-core/src/main/kotlin/arrow/core/Option.kt

```diff
@@ -325,6 +328,7 @@ import arrow.typeclasses.Semigroup
  * Contents partially adapted from [Scala Exercises Option Tutorial](https://www.scala-exercises.org/std_lib/options)
  * Originally based on the Scala Koans.
  */
+@OptIn(ExperimentalContracts::class)
```

Comment:
We shouldn't make `Option` `Experimental`. Are contracts still experimental?

Reply:
I really don't want this as well. I'll test adding the ExperimentalContracts annotation to each method instead and check from a different lib like fx if the compiler still complains there. But it won't work without the annotation, intellij and the compiler both complain.

Reply:
Per method annotations force me to add this to every method that calls a annotated one. This isn't nice, but yeah I kind of think they should not be experimental as well. Maybe a old kotlin version? Will check more tomorrow.